.gitignore

```diff
@@ -10,3 +10,5 @@ tmp
 .classpath
 .project
 .settings
+*.swp
+tags
```

Comment:
You're right – .DSStore, .classpath, .idea (and more?) shouldn't be here either 😄